forms-flow-web/src/apiManager/services/processServices.js

```diff
@@ -66,10 +66,12 @@ export const getProcessStatusList = (processId, taskId) => {
 export const fetchAllBpmProcesses = (...rest) => {
   const done = rest.length ? rest[0] : () => {};
   return (dispatch) => {
-    httpGETRequest(API.PROCESSES, {}, UserService.getToken(), true)
+    httpGETRequest(API.GET_BPM_PROCESS_LIST + "?latestVersion=true&excludeInternal=true", {}, UserService.getToken(), true)
       .then((res) => {
         if (res.data) {
-          dispatch(setAllProcessList(res.data.process));
+          if(res.data._embedded.processDefinitionDtoList) {
```

Comment:
check for res.data && res.data._embedded or res?.data?._embedded?
.processDefinitionDtoList

Reply:
if there are no process definitions then res.data can come without _embedded. If _embedded is there then always processDefinitionDtoList also will be there